fe/fe-core/src/test/java/org/apache/doris/nereids/rules/analysis/GenerateFunction.java

```diff
@@ -95,6 +95,7 @@
 import java.util.function.Supplier;
 import java.util.stream.Collectors;
 
+@Deprecated
```

Comment:
why add this?

Reply:
If I understand correctly, this class has not been maintained for several years, but each Function definition class has a line `This class is generated by GenerateFunction`, like [`Histogram`](https://github.com/apache/doris/blob/master/fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/agg/Histogram.java). Maybe this is a kind of misleadingness, especially for new guys. So I add the `@Deprecated`. If this is not correct, I will remove it.

Reply:
yes, it is used for batch generate Nereids' functions from legacy planner's function when we develop Nereids at very beginning. add `@Deprecated` is ok